ajax/libs/layui/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "layui",
+  "filename": "layui.js",
+  "version": "2.2.45",
+  "description": "经典模块化前端框架",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/sentsin/layui.git"
+  },
+  "author": "贤心",
+  "homepage": "http://www.layui.com",
+  "keywords": [
+    "layui",
+    "ui"
```

Comment:
(Optional) More keywords may help users find this library.

Reply:
okay thanks for suggest